music_assistant/providers/subsonic_scrobble/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
why not add "depends_on": "subsonic" to the manifest ?

Reply:
Wasn't aware of that. Makes much more sense 👍